### PR TITLE
Add Full Width prop to PlmgCard component

### PR DESCRIPTION
**Pull request by JakeH91**
helps with [this](https://app.asana.com/0/1198902771377180/1205850611994929)

### Summary of changes included in this PR
- Adds full width to the card so it can fill the width of it's parent, rather than be the width of it's content (either is an option now)

### Reviewer checklist:

- Tests for
  - accessibility
  - rendered HTML (spec)
- Storybook stories for all variants
- JSDoc documentation for component
- Update Example app (React, ...)

### CODE REVIEW

File: packages/component-library/src/components.d.ts [hunk 1]
```diff
@@ -111,6 +111,10 @@ export namespace Components {
           * Define card's bottom button text.  If a text is provided, the button will be displayed. By default, when no text is provided, the button is hidden.
          */
         "bottomActionText": string | undefined;
+        /**
+          * Define card's sizing behaviour. To stretch to it's content, or fill the available space it's in.  If fullWidth is provided, the card will take the full width of it's container. By default, when no fullWidth is provided, it will be the size of it's contents (plus it's padding).   Allowed values:   - true   - false  Default: false
```
Comment (jamesmarriott) [inline packages/component-library/src/components.d.ts:115]:
```suggestion
          * Define card's sizing behaviour. To stretch to its content, or fill the available space its in.  If `fullWidth` is provided, the card will take the full width of it's container.  If `fullWidth` is not specified, the card will automatically size itself according to its content, plus any padding.   Allowed values:   - true   - false  Default: false
```

File: packages/component-library/src/components/plmg-card/plmg-card.tsx [hunk 2]
```diff
@@ -79,7 +99,10 @@ export class Card {
     };
 
     return (
-      <div class={'plmg-card'}>
+      <div
+        class={'plmg-card'}
+        style={{ width: this.fullWidth ? 'full-width' : 'fit-content' }}
```
Comment (jamesmarriott) [inline packages/component-library/src/components/plmg-card/plmg-card.tsx:104]: width: full-width is not valid css. It's just ignoring this rule.  Could just write `style={{ width: !this.fullWidth && 'fit-content' }}`?

**Review by jamesmarriott — CHANGES_REQUESTED** [review]

File: packages/component-library/src/components/plmg-card/plmg-card.tsx (hunk 2 above)
Comment (JakeH91) [reply packages/component-library/src/components/plmg-card/plmg-card.tsx:104]: Yeah I noticed that actually, was just copying PlmgButton's implementation, but I will adjust here anyway as you are right it is doing nothing lol.

**Review by jamesmarriott — APPROVED** [review]
Nice! Let's see if deploys

**Review by jamesmarriott — APPROVED** [review]

**Review by jamesmarriott — APPROVED** [review]

**Review by jamesmarriott — APPROVED** [review]

**Review by jamesmarriott — APPROVED** [review]

**Review by jamesducky — APPROVED** [review]

**Pull request merged**